.circleci/config.yml

```diff
@@ -8,6 +8,9 @@ defaults: &defaults
     - EMSDK_NUM_CORES: 3
       EMCC_CORES: 3
       PYODIDE_JOBS: 3
+      # Make sure the ccache dir is consistent between core and package builds
+      # (it's not the case otherwise)
```

Comment:
Maybe something to do with emsdk activate, by it in any case we were previously caching the wrong path for packages